explore/src/main/scala/explore/config/ConfigurationPanel.scala

```diff
@@ -89,52 +92,58 @@ object ConfigurationPanel {
   protected val component =
     ScalaFnComponent
       .withHooks[Props]
+      .useStateSnapshotWithReuse(none[ITCRequestsQueue[IO]])
+      .useEffectOnMountBy { (props, st) =>
+        // Create the requests queue and put it on the state
+        implicit val ctx: AppContextIO = props.ctx
+        // TODO proper cleanup
+        ITCRequestsQueue.build[IO]((r: Option[ITCRequestsQueue[IO]]) => st.setState(r).to[IO])
```

Comment:
Maybe we can separate concerns by having `build` return a `F[ITCRequestsQueue]` and then flatMap to `setState` locally ?